cmake/thisdd4hep.sh

```diff
@@ -61,7 +65,12 @@ dd4hep_add_library_path()    {
 }
 #-----------------------------------------------------------------------------
 #
-dd4hep_parse_this ${BASH_ARGV[0]} DD4hep;
+SOURCE=${BASH_ARGV[0]}
+if [ "x$SOURCE" = "x" ]; then
+    SOURCE=${(%):-%N} # for zsh
+fi
+
+dd4hep_parse_this $SOURCE  @PackageName@;
```

Comment:
```suggestion
dd4hep_parse_this $SOURCE DD4hep;
```
As it was before

Reply:
but that was probably wrong, it should at best be not capitalized just `dd4hep`